pkgs/tools/security/ssh-audit/default.nix

```diff
@@ -1,23 +1,53 @@
-{ lib, fetchFromGitHub, python3Packages }:
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
 
-python3Packages.buildPythonApplication rec {
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      click = super.click.overridePythonAttrs (oldAttrs: rec {
+        version = "7.1.2";
+        src = oldAttrs.src.override {
+          inherit version;
+          hash = "02kbzd6sjfkqan3miwj1wqyddfxc2b6hi7p5s4dvqjb3gif2bdfj";
+        };
+      });
+    };
+  };
+in
+with py.pkgs;
```

Comment:
Not a fan of bringing the whole package set into the scope for the whole derivation.